css/variables.css

```diff
@@ -0,0 +1,41 @@
+:root{
+    /* sleek ui colors */
+    --primary-color: #D42142;
+
+    /* action colors */
+    --success-color: #52C41A;
+    --warning-color: #FAAD14;
+    --danger-color: #FF4D4F;
+    --info-color: #CFF4FC;
+    --light-color: #F8F9FA;
+    --dark-color: #212529;
+
+    /* Basic colors */
+    --black-color: #282828;
+    --white-color: #FFF;
+    --lynx-white-color: #F7F7F7;
+    --gray-color: #9A9A9A;
+    --lynx-gray-color: #C5C5C5;
+
+    /* Font sizes */
+    --font-size-1: 4.8rem;
+    --font-size-2: 3.2rem;
+    --font-size-3: 2.4rem;
+    --font-size-4: 1.6rem;
+    --font-size-5: 1.2rem;
+    --font-size-6: 1rem;
+
+    /* line height */
+    --line-heignt-1: 6.4rem;
+    --line-heignt-2: 4rem;
+    --line-heignt-3: 3.2rem;
+    --line-heignt-4: 2.4rem;
+    --line-heignt-5: 1.6rem;
+    --line-height-6: 1.2rem;
+
+    /* Border Radius */
+    --border-radius-small: 5px;
+    --border-radius-medium: 10px;
+    --border-radius-larger: 15px;
+    --border-radius-circle: 999px;
```

Comment:
you can also use "br" for class name for "border-radius". saves a lot of space!

Reply:
Changes the classes